File: 2942-find-words-containing-character/2942-find-words-containing-character.cs
public class Solution {
    public IList<int> FindWordsContaining(string[] words, char x) {
        List<int> index = new List<int>();
        int n = words.Length;
        for (int i = 0; i < n; i++)
        {
            if (words[i].IndexOf(x) != -1)
            {
                index.Add(i);
            }
        }
        
        return index;
    }
}

File: 2942-find-words-containing-character/2942-find-words-containing-character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int>index;
        int n = words.size();
        for(int i=0; i<n; i++)
        {
            if (words[i].find(x) != string::npos) 
            {
                index.push_back(i);
            }
        }
        return index;
    }
};